crates/nu-command/src/commands/run_external.rs

```diff
@@ -101,25 +96,6 @@ impl WholeStreamCommand for RunExternalCommand {
             },
         };
 
-        // If we're in interactive mode, we will "auto cd". That is, instead of interpreting
-        // this as an external command, we will see it as a path and `cd` into it.
-        if is_interactive {
-            if let Some(path) = maybe_autocd_dir(&command, &mut external_context) {
-                let cd_args = CdArgs {
```

Comment:
This code is removed and the case is handled in cd. 

(Actually I strongly believe and tested that maybe_autocd_dir never returns Some(...), as this case was handled in script.rs before anyways.)